app/assets/javascripts/govuk/selection-buttons.js

```diff
@@ -6,8 +6,6 @@
   if (typeof GOVUK === 'undefined') { root.GOVUK = {}; }
 
   var SelectionButtons = function (elmsOrSelector, opts) {
-    var $elms;
```

Comment:
yeh you're right - looks like we ought to be able to refer to it in the toolkit that gets downloaded, either with this line or something similar: https://github.com/alphagov/govuk_prototype_kit/blob/master/server.js#L58